example/package.json

```diff
@@ -44,5 +44,11 @@
   ],
   "files": [
     "build"
-  ]
+  ],
+  "jest": {
```

Comment:
I think this change may already be present in `develop`

Reply:
Is this dependency needed in `example`? From what I understood from the issue, we're altering `src/component`s with markdown. Are we altering anything with markdown in `example`?

Reply:
We aren't altering anything with markdown in `example`, however in one of my previous commits (d17d0a4), the following error is displayed when trying to build and test:
![Screenshot (10)](https://user-images.githubusercontent.com/81392411/183511889-ac39c81c-fdc9-42f3-95c4-01ddc203aa9f.png)
This is the same error I was getting when trying to run the component tests locally before adding the dependency in `package.json`. Adding the same dependency in `example/package.json` fixed the error as well. You are correct that nothing is being altered with markdown in `example`, so I am confused as to why this is necessary.